src/server/scripts/Outland/zone_shattrath_city.cpp

```diff
@@ -86,18 +86,26 @@ class npc_raliq_the_drunk : public CreatureScript
         {
             m_uiNormFaction = creature->GetFaction();
         }
-
+        uint32 Drink_Timer = 5000;
```

Comment:
use eventScheduler instead

Reply:
Using TaskScheduler now. I am not quite sure about line 118 in terms of syntax - it works but I'm not sure how you'd like it written.

Reply:
Now using TaskScheduler